webapp/src/components/ProfilePage/Contribution/Contribution.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { Card, Header, Button } from 'semantic-ui-react'
+import { contributionType } from 'components/types'
+
+import './Contribution.css'
+
+export default class Publication extends React.PureComponent {
+  static propTypes = {
+    contribution: contributionType
+  }
+  render() {
+    const { contribution } = this.props
+
+    return (
+      <Card className="Contribution">
+        <Card.Content className="body">
+          <Header size="medium">
+            {contribution.district ? contribution.district.name : 'Loading...'}
+          </Header>
+          <Card.Meta>
+            {contribution.district
+              ? contribution.district.description
+              : 'Loading...'}
+          </Card.Meta>
+        </Card.Content>
+        <Card.Content extra>
+          <span className="footer">
+            <Header size="medium" floated="left" className="land-count">
+              <span className="amount">{contribution.land_count}</span>{' '}
+              &nbsp;LAND
+            </Header>
+            <a
+              href={contribution.district ? contribution.district.link : '#'}
+              target="_blank"
+              rel="noopener noreferrer"
+            >
```

Comment:
I think this may be confusing. Even if the contribution says loading, clicking a link and getting a `#` it's never nice.

What do you think about doing something like:

```javascript
{contribution.district ? (
  <a
    href={contribution.district.link}
    target="_blank"
    rel="noopener noreferrer"
  >
    <Button floated="right" size="tiny">
      Proposal
    </Button>
  </a>
) : (
  <Button floated="right" size="tiny" disabled={true}>
    Proposal
  </Button>
)}
```

(not tested, wrote here)